test/end-to-end/fee_center/feeCenterCU.page.js

```diff
@@ -0,0 +1,126 @@
+
+/* jshint expr:true */
+/* global element, by, browser */
+
+/**
+ * This class is represents a fee center creation page in terms of structure and
+ * behaviour so it is a fee center creation page object
+ **/
+function CreateUpdateFeeCenterPage() {
```

Comment:
In my view, the page object methods works best on really complicated pages - such as Complex Vouchers, Patient Invoice, Posting Journal...

Your current page is a simple form - in this case, it triples the amount of code needed to maintain and upgrade with marginal benefit.